llvm/include/llvm/CodeGen/FunctionLoweringInfo.h

```diff
@@ -73,8 +73,8 @@ class FunctionLoweringInfo {
   /// allocated to hold a pointer to the hidden sret parameter.
   Register DemoteRegister;
 
-  /// MBBMap - A mapping from LLVM basic blocks to their machine code entry.
-  DenseMap<const BasicBlock*, MachineBasicBlock *> MBBMap;
+  /// MBBMap - A mapping from LLVM basic block number to their machine block.
```

Comment:
https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments says "Don’t duplicate function or class name at the beginning of the comment."

As you are updating the comment, drop `MBBMap - `